python/sagemaker-multumodel-endpoint/examples/download_models.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import os
+import os.path
+import tarfile
+import shutil
+
+resnet_18_params = (
+    "http://data.mxnet.io/models/imagenet/resnet/18-layers/resnet-18-0000.params"
+)
+resnet_18_symbols = (
+    "http://data.mxnet.io/models/imagenet/resnet/18-layers/resnet-18-symbol.json"
+)
+
+resnet_15_params = (
+    "http://data.mxnet.io/models/imagenet/resnet/152-layers/resnet-152-0000.params"
+)
+resnet_15_symbols = (
+    "http://data.mxnet.io/models/imagenet/resnet/152-layers/resnet-152-symbol.json"
+)
+
+synset = "http://data.mxnet.io/models/imagenet/synset.txt"
```

Comment:
Let's note this URL in the README so that users understand that this will require sending requests to a 3rd party.

Reply:
Added README with explanation